setup.py

```diff
@@ -60,9 +60,9 @@ def run(self):
 
 requirements = galaxy_pulp_requirements + [
     "Django~=2.2.3",
-    "pulpcore @ git+https://github.com/pulp/pulpcore.git@master#egg=pulpcore",
-    "pulp_ansible @ git+https://github.com/pulp/pulp_ansible.git@master#egg=pulp_ansible",
-    "pulp_container @ git+https://github.com/pulp/pulp_container.git@master#egg=pulp_container",
+    "pulpcore==3.6.0",
```

Comment:
Please use loose version restrictions in `setup.py`. Pulp uses semantic versioning, so it must be safe to use Compatible Release Specifiers, like: 

```
pulpcore~=3.6.0
pulp-ansible~=0.2.0
pulp-container~=2.0.0
```

Reply:
pulp-container is not a direct dependency of `galaxy_ng`. It must not be a part of `setup.py`.

Reply:
Fixed.